python/lsst/dax/imgserv/vo/imageSODA.py

```diff
@@ -85,6 +84,21 @@ def do_async(self, params: dict) -> object:
         """
         raise NotImplementedError("ImageSODA.do_async()")
 
+    def do_sia(self, params: dict) -> object:
+        """ Do async operation.
+
+        Parameters
+        ----------
+        params : `dict`
+            the HTTP parameters.
+
+        Returns
+        -------
+        xml: `str`
+
+        """
+        raise NotImplementedError("ImageSODA.do_sia()")
```

Comment:
So you're adding in SIA?  Or is this future work that will be done soon?

Reply:
See above comment.